Dockerfile

```diff
@@ -0,0 +1,80 @@
+FROM composer:2.0 as composer
+
+ARG TESTING=false
+ENV TESTING=$TESTING
```

Comment:
We dont use these args. Lets remove them

Reply:
Fixed.